Sources/MusicNotationKit/FileIO/Protobuf/_Interval+Transform.swift

```diff
@@ -0,0 +1,19 @@
+import MusicNotationCore
+
+extension _Interval {
+    func transformQuality() -> MusicNotationCore.IntervalQuality {
+        switch self.quality {
+        case .augmented: return MusicNotationCore.IntervalQuality.augmented
+        case .diminished: return MusicNotationCore.IntervalQuality.diminished
+        case .doublyAugmented: return MusicNotationCore.IntervalQuality.doublyAugmented
+        case .doublyDiminished: return MusicNotationCore.IntervalQuality.doublyDiminished
+        case .major: return MusicNotationCore.IntervalQuality.major
+        case .minor: return MusicNotationCore.IntervalQuality.minor
+        case .perfect: return MusicNotationCore.IntervalQuality.perfect
+        default: return MusicNotationCore.IntervalQuality.major
```

Comment:
Since you declare the return type, you should be able to just put the short version, like `.augmented'.